tests/suites/test_suite_ssl.function

```diff
@@ -1997,7 +2051,11 @@ void perform_handshake( handshake_test_options* options )
     else
     {
         TEST_ASSERT( mbedtls_endpoint_init( &server, MBEDTLS_SSL_IS_SERVER,
-                                            options->pk_alg, NULL, NULL,
+                                            options->pk_alg,
```

Comment:
At this point it might be cleaner to just pass the options.

Reply:
I would love to but `mbedtls_endpoint_init()` is also used directly.